packages/common/src/ConfigResponse.ts

```diff
@@ -0,0 +1,59 @@
+import {
+  Address,
+  IntString,
+  NpmLogLevel
+} from './types/Aliases'
+import { Environment } from './Environments'
+
+export interface LoggerConfiguration {
+  logLevel: NpmLogLevel
+  loggerUrl?: string
+  userId?: string
+  applicationId?: string
+}
+
+export interface ConfigResponse {
+  networks: Record<number|string, ConfigEntry>
+}
+
+export interface ConfigEntry {
+  name: string
+  gsnConfig: Partial<GSNConfig>
+}
+
+/**
+ * @field methodSuffix - allows use of versioned methods, i.e. 'eth_signTypedData_v4'. Should be '_v4' for Metamask
+ * @field jsonStringifyRequest - should be 'true' for Metamask, false for ganache
```

Comment:
As you are modifying it, let's either have all comments up here or all comments down where the variable is located, but not both.

Reply:
I only moved that part from GSNConfigurator.ts

Reply:
fixed